src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -184,6 +186,7 @@ public void OnEventWritten_New(string name, object payload)
         TagList tags = default;
 
         // see the spec https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/http/http-spans.md
+        tags.Add(new KeyValuePair<string, object>(SemanticConventions.AttributeNetworkProtocolName, NetworkProtocolName));
```

Comment:
is this related to .net 8.0?

Reply:
it is not directly related to it. But this change ensures matching attributes between lower versions and .net8.0. Missed adding this to changelog. Will update.

Reply:
updated the changelog